pylint/checkers/utils.py

```diff
@@ -1950,6 +1950,8 @@ def is_hashable(node: nodes.NodeNG) -> bool:
         for inferred in node.infer():
             if inferred is astroid.Uninferable:
                 return True
+            if not hasattr(inferred, "igetattr"):
```

Comment:
`mypy` will complain about this when we add `py.typed` to `astroid`, but that is a problem to fix then I think.